src/nova_tz/rsa_key.rs

```diff
@@ -0,0 +1,75 @@
+use crate::nova_tz::keyblob;
+use crate::nova_tz::Error;
+use crate::nova_tz::Result;
+use dynfmt::{Format, SimpleCurlyFormat};
+use std::convert::TryFrom;
+use std::fs;
+use std::path::Path;
+use uuid::Uuid;
+
+pub struct RsaKey {
```

Comment:
This name confused me a lot.. it's not an RSAKey but, what is it? A TZKeypair registry?